packages/icon-builder/package.json

```diff
@@ -24,20 +24,20 @@
   "author": "Hai Nguyen",
   "license": "MIT",
   "dependencies": {
-    "yargs": "^3.15.0"
+    "yargs": "^4.3.2"
   },
   "peerDependencies": {},
   "devDependencies": {
     "babel-cli": "^6.4.5",
     "chai": "^3.0.0",
-    "glob": "^5.0.14",
+    "glob": "^7.0.3",
```

Comment:
We have `glob`, `babel-cli`, `chai`, `mocha``, etc... in the main repo. Do they need to be in here too?

(Side note, I'll remove grunt at some point too and just make it vanilla node)